vars/runGradleCheck.groovy

```diff
@@ -51,7 +51,7 @@ void call(Map args = [:]) {
                 ./gradlew --stop
                 rm -rf ~/.gradle
 
-                if ! (uname -s | grep -i NT); then
+                if command -v docker > /dev/null; then
                     echo "Check existing dockercontainer"
                     docker ps -a
```

Comment:
`ps -a` would also list the exited containers which are already stopped. 
If we just want to stop the running containers then we should remove the `-a` flag. 
@peterzhuamazon